todo_app/data/session_items.py

```diff
@@ -38,32 +53,26 @@ def add_item(title):
         title: The title of the item.
 
     Returns:
-        item: The saved item.
+        boolean: A boolean denoting whether the function succeeded or not.
     """
-    items = get_items()
-
-    # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
-
-    item = { 'id': id, 'title': title, 'status': 'Not Started' }
-
-    # Add the item to the list
-    items.append(item)
-    session['items'] = items
-
-    return item
+    responseBody = post(_CARDS_BASE_URL, params={ **_BASE_PARAMS, 'idList':_BASE_LIST_ID + _NOT_STARTED_LIST_ID, 'name': title })
```

Comment:
I suspect you've done some refactoring; these variables aren't available in this file

Reply:
Resolved in commit 4f9b126